tests/functional/build/test_build.py

```diff
@@ -0,0 +1,192 @@
+import pytest
+
+from dbt.tests.util import run_dbt, copy_file
+
+
+from tests.functional.build.fixtures import (  # noqa: F401
+    snapshots,
+    tests_failing,
+    models,
+    models_failing,
+    seeds,
+    models_circular_relationship,
+    models_simple_blocking,
+    test_files,
+    models_interdependent,
+    project_files,
+)
+
+
+# from test.integration.base import DBTIntegrationTest, use_profile, normalize
+from test.integration.base import normalize
+import yaml
+import os
+
+
+class TestBuildBase:
+    @pytest.fixture(scope="class")
+    def project_config_update(self):
+        return {
+            "config-version": 2,
+            "snapshot-paths": ["snapshots"],
+            "seed-paths": ["seeds"],
+            "seeds": {
+                "quote_columns": False,
+            },
+        }
```

Comment:
This isn't necessary unless you need to change from the default config.  Since we no longer use the complex mult-folder structure to store files for testing, most of the time this doesn't need to be touched at all.  Since the default for quote_columns is False, you can remove this entirely.  This is true in all other tests that I can tell.  Some of them are pointing to use non-default directories but I don't think it's strictly necessary or what we're actually testing here.

Reply:
Sure thing.  @ChenyuLInx , same as above.

Reply:
We could add a process function [here](https://github.com/ChenyuLInx/unittest2pytest/blob/dbt-pytest-all-dir/unittest2pytest/fixes/fix_functions.py#L130) to update that node? But manipulating and checking the dict structure with the framework is not super easy so I vote for scan through case by case, I will document this in the conversion guide.